app/controllers/posts_controller.rb

```diff
@@ -160,6 +160,67 @@ def show
 
   def edit; end
 
+  # Attempts to update a given post
+  # @param post [Post] post the user is attempting to update
+  # @param user [User] user attempting to update the post
+  # @param edit_post_params [ActionController::Parameters] edit parameters
+  # @param body_rendered [String] new post body
+  # @return [Boolean] status of the operation
+  def do_update(post, user, edit_post_params, body_rendered)
```

Comment:
suggestion: I'd refactor this signature to the following for Ruby conventions:

```suggestion
  def do_update(post, user, body_rendered, **edit_post_params)
```

This means you can call it as `do_update(post, user, body, some: 'data', more: 'data')` instead of `do_update(post, user, { some: 'data', more: 'data' }, body)`. Little bit nicer.

Reply:
Yeah, let's do that (same with the `do_update_network`), keep forgetting the double splat operator exists, a moment.